packages/kbn-logging/src/log_meta.ts

```diff
@@ -6,40 +6,41 @@
  * Side Public License, v 1.
  */
 
-import { EcsBase } from './ecs/base';
-
-import { EcsAgent } from './ecs/agent';
-import { EcsAutonomousSystem } from './ecs/autonomous_system';
-import { EcsClient } from './ecs/client';
-import { EcsCloud } from './ecs/cloud';
-import { EcsContainer } from './ecs/container';
-import { EcsDestination } from './ecs/destination';
-import { EcsDns } from './ecs/dns';
-import { EcsError } from './ecs/error';
-import { EcsEvent } from './ecs/event';
-import { EcsFile } from './ecs/file';
-import { EcsGroup } from './ecs/group';
-import { EcsHost } from './ecs/host';
-import { EcsHttp } from './ecs/http';
-import { EcsLog } from './ecs/log';
-import { EcsNetwork } from './ecs/network';
-import { EcsObserver } from './ecs/observer';
-import { EcsOrganization } from './ecs/organization';
-import { EcsPackage } from './ecs/package';
-import { EcsProcess } from './ecs/process';
-import { EcsRegistry } from './ecs/registry';
-import { EcsRelated } from './ecs/related';
-import { EcsRule } from './ecs/rule';
-import { EcsServer } from './ecs/server';
-import { EcsService } from './ecs/service';
-import { EcsSource } from './ecs/source';
-import { EcsThreat } from './ecs/threat';
-import { EcsTls } from './ecs/tls';
-import { EcsTracing } from './ecs/tracing';
-import { EcsUrl } from './ecs/url';
-import { EcsUser } from './ecs/user';
-import { EcsUserAgent } from './ecs/user_agent';
-import { EcsVulnerability } from './ecs/vulnerability';
+import {
+  EcsBase,
+  EcsAgent,
+  EcsAutonomousSystem,
+  EcsClient,
+  EcsCloud,
+  EcsContainer,
+  EcsDestination,
+  EcsDns,
+  EcsError,
+  EcsEvent,
+  EcsFile,
+  EcsGroup,
+  EcsHost,
+  EcsHttp,
+  EcsLog,
+  EcsNetwork,
+  EcsObserver,
+  EcsOrganization,
+  EcsPackage,
+  EcsProcess,
+  EcsRegistry,
+  EcsRelated,
+  EcsRule,
+  EcsServer,
+  EcsService,
+  EcsSource,
+  EcsThreat,
+  EcsTls,
+  EcsTracing,
+  EcsUrl,
+  EcsUser,
+  EcsUserAgent,
+  EcsVulnerability,
+} from '@kbn/ecs';
```

Comment:
<img width="1072" alt="Screen Shot 2022-12-12 at 7 43 18 PM" src="https://user-images.githubusercontent.com/6935300/207213623-0b4e222a-8cff-4b2d-af71-7b7f83ef6499.png">

how do i need to define this besides the definitions I've put into  `kbn-logging/BUILD.bazel`?

Reply:
yay idk what i did but webstorm stopped complaining!